drivers/SmartThings/philips-hue/src/utils.lua

```diff
@@ -54,6 +54,23 @@ function utils.kelvin_to_mirek(kelvin) return 1000000 / kelvin end
 
 function utils.mirek_to_kelvin(mirek) return 1000000 / mirek end
 
+-- TODO: The Hue API itself doesn't have events for multipresses, however, it will
+-- emit batched "short release" eventsource on the SSE stream if they're close together.
+-- Right now the SSE stream handling is a relatively dumb pass-through that doesn't inspect
+-- the data as it unpacks it; it just dispatches each event in the batch as it encounters it.
+-- We could implement 2x-6x press if we add some smarts to the SSE stream handling.
```

Comment:
Possibly, I wouldn't try to force it. There's ways to do this with just the pass-through (involving lots of timers), but I'd say if it's not distinguished in the API then you've made the right call.